exec/cnex/array.c

```diff
@@ -83,9 +95,13 @@ void array_clearArray(Array *self)
         for (size_t i = 0; i < self->size; i++) {
             cell_clearCell(&self->data[i]);
         }
-        free(self->data);
-        self->data = NULL;
+        // ...Wait.  Why are we freeing the array here?  Why bother?  Just leave it, with self->max set, and
```

Comment:
This seems to be a "note to self" comment that no longer needs to be present. The code it's referring to is not present.

Reply:
Removed.